opm/output/data/CellDataContainer.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <algorithm>
+
+#include <opm/output/data/CellDataContainer.hpp>
+#include <opm/output/data/Cells.hpp>
+
+
+namespace Opm {
+
+    CellDataContainer::CellDataContainer( std::initializer_list<data::CellData> init_list )
+        : data( init_list )
+    { }
+
+
+    CellDataContainer::CellDataContainer( std::vector<data::CellData> init_list )
+        : data(std::move(init_list))
```

Comment:
Out of curiosity: Has anybody checked what the compiler does here? I would assume that it has to do the copy and hopefully is smart enough to not move. I any case it seem that this will never be better than
```
CellDataContainer::CellDataContainer( const std::vector<data::CellData>& init_list )
        : data(init_list)
```
Which does definitely just copy. And if init_list is a just created temporary then the compiler might be smart enough to do just the construction and skip the copy.